CentralHub.Api.Model/Measurement.cs

```diff
@@ -0,0 +1,26 @@
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+
+namespace CentralHub.Api.Model;
+
+public sealed class Measurement
+{
+    [Obsolete("Deserialization only")]
+    public Measurement()
+    {
+    }
+
+    public Measurement(string type, string addr, int measurement)
+    {
+        MacAddress = addr;
+        Type = type;
+        Value = measurement;
+    }
+
+    public string MacAddress { get; set; }
+
+    public string Type { get; set; }
+
+    public int Value { get; set; }
```

Comment:
it cannot be called Measurement, as that name is taken by the class. Otherwise another name for the class?

I also don't like accessing it as Measurement.Measurement, Measurement.Value looks better to me

Reply:
But what is a measurement? RSSI? If that is the case call it RSSI